src/content/contributing/translation-program/blog-translations/index.md

```diff
@@ -22,7 +22,7 @@ Anyone can contribute to translating the EF blog. If you want to get involved, k
 
 </InfoBanner>
 
-### Join the blog translations project in Crowdin {#join-crowdin}
+### Join the blog translations project in Crowdin {#join-crowdin}
```

Comment:
@wackerow do we know why this happens? its quite annoying 😅

Reply:
Looks like for it to be a MDX heading, it must use the 'space' character (32) and not the 'non-breaking-space' char (160).

<img width="663" alt="Screenshot 2022-08-25 at 09 17 48" src="https://user-images.githubusercontent.com/62268199/186613054-514abc36-5036-478b-90a9-aaf0ca18af1e.png">

Still not sure why sometimes we get a nbsp instead of a regular space... 🤔 

Can we overwrite this behaviour? i.e. #[...] + nbsp also is a heading?